src/shogun/classifier/svm/LibLinear.cpp

```diff
@@ -95,8 +93,14 @@ bool CLibLinear::train_machine(CFeatures* data)
 	int32_t num_feat=features->get_dim_feature_space();
 	int32_t num_vec=features->get_num_vectors();
 
-	if (liblinear_solver_type == L1R_L2LOSS_SVC ||
-			(liblinear_solver_type == L1R_LR) )
+	// rename those names to aviod the shadow declaration
+	bool enable_bias = get_bias_enabled();
```

Comment:
all of those dont have to be in a variable, no?
@lisitsyn we discussed this in irc